packages/@o3r/core/schematics/component/presenter/index.ts

```diff
@@ -130,6 +130,10 @@ export function ngGenerateComponentPresenter(options: NgGenerateComponentSchemat
       // Angular schematics generate style file with this pattern: component-name.component.scss
       chain([
         move(ngStylePath, o3rStylePath),
+        (t) => {
+          // Styling file is empty by default, as we create component with `viewEncapsulation` set to 'None', we should wrap the styling into the selector of the component
+          t.overwrite(o3rStylePath, `${properties.componentSelector}-${properties.suffix} {\n\t// Your component custom SCSS\n}\n`);
```

Comment:
should return `t`

Reply:
https://github.com/AmadeusITGroup/otter/pull/733
then we need to fix it on release/9.1 :)

Reply:
https://github.com/AmadeusITGroup/otter/pull/741